lib/analyze/collect/source.js

```diff
@@ -34,6 +34,7 @@ function inspectFiles(data, dir) {
         testsSize: detectRepoTestFiles(dir).then((files) => fileSize(files)),
         hasNpmIgnore: isRegularFile(`${dir}/.npmignore`).then((is) => is || null),
         hasShrinkwrap: isRegularFile(`${dir}/npm-shrinkwrap.json`).then((is) => is || null),
+        hasTypes: isRegularFile(path.join(dir, data.typings || data.types || 'index.d.ts')).then((is) => is || null),
```

Comment:
Everything looks great, except the `hasTypes` which is too generic and doesn't resemble typescript. Can you change to something more explicit?